pkg/fanal/types/misconf.go

```diff
@@ -67,7 +67,8 @@ type Line struct {
 }
 
 type PolicyMetadata struct {
-	ID                 string   `json:",omitempty"`
+	ID string `json:",omitempty"`
```

Comment:
I think we can mark this field as deprecated in the `rpc` package:
https://github.com/aquasecurity/trivy/blob/aeeb2a1f842b56147996b600bd34db2cf05cd28e/rpc/common/service.proto#L93